StabilityPackage.wl

```diff
@@ -5,22 +5,37 @@ BeginPackage["StabililtyPackage`"]
 ImportData::usage =
 	"ImportData[file] reads in the data from file."
 buildHamiltonians::usage=
-	"Return elements of the hamiltonians"
+	"[data,test Energy, Hierarchy=1/-1]. Return elements of the hamiltonians"
 getEquations::usage=
-	"Return the equations of motions"
-rules::usage=
-	"Some substitution rules"
+	"[data,test energy, hierarchy=-1/1, k]. Return the equations of motions"
 stabilityMatrix::usage=
-	"returns the stability matrix"
-evscale::usage=
-	"solves for the eigenvalues"
-SCalcScale::usage=
-	"Computes several things"
+	"[data,Equations of motion (getEquations output)]. Returns the stability matrix"
 buildkGrid::usage=
-	"constructions the adaptive k grid"
+"[data, number of k steps, (optional) manual target k, 
+(optional) k lowwer bound multiple; Default \!\(\*SuperscriptBox[\(10\), \(-3\)]\),(optional) k upper bound Default: 10]. Creates the k grid"
 kAdapt::usage=
-	"runs the full routine"
-	
+	"[file, radius start, radius end, test energy, hierarchy, number of k steps, (optional) manual target k, 
+(optional) k lowwer bound multiple; Default \!\(\*SuperscriptBox[\(10\), \(-3\)]\),(optional) k upper bound Default: 10]. Runs the full routine"
+GDValue::usage=
+	"returns the maximum possible value of the frequency"
+GDdata::usage=
```

Comment:
If it's exactly the same, I think the filename should reflect that. E.g. kadaptGD[]